pkg/types/version.go

```diff
@@ -0,0 +1,64 @@
+package types
+
+import (
+	"flag"
+	"fmt"
+	"runtime/debug"
+	"strings"
+)
+
+var (
+	// set using the '-X github.com/containers/gvisor-tap-vsock/pkg/types.ModuleVersion' linker flag
+	gitVersion = ""
+	// set through .gitattributes when `git archive` is used
+	// see https://icinga.com/blog/2022/05/25/embedding-git-commit-information-in-go-binaries/
+	gitArchiveVersion = "$Format:%(describe)$"
```

Comment:
This convert to  `git describe`  one right or `git log --pretty=format"$Format:%(describe)$"` one?

Reply:
The documentation for `git log --pretty=format"$Format:%(describe)"$` says 
> `%(describe[:<options>])`
>               human-readable name, like `git-describe(1)`;

so they are both the same?